cpp/src/dataframe/column.cpp

```diff
@@ -34,24 +34,7 @@ gdf_error gdf_mask_concat(gdf_valid_type *output_mask,
                           gdf_size_type *column_lengths, 
                           gdf_size_type num_columns);
 
-/** ---------------------------------------------------------------------------*
- * @brief Concatenates multiple gdf_columns into a single, contiguous column,
- * including the validity bitmasks.
- * 
- * Note that input columns with nullptr validity masks are treated as if all
- * elements are valid.
- *
- * @param[out] output_column A column whose buffers are already allocated that
- *             will contain the concatenation of the input columns data and
- *             validity bitmasks
- * @param[in] columns_to_concat[] The columns to concatenate
- * @param[in] num_columns The number of columns to concatenate
- * 
- * @return gdf_error GDF_SUCCESS upon completion; GDF_DATASET_EMPTY if any data
- *         pointer is NULL, GDF_COLUMN_SIZE_MISMATCH if the output column size
- *         != the total size of the input columns; GDF_DTYPE_MISMATCH if the
- *         input columns have different datatypes.
- * ---------------------------------------------------------------------------**/
+
```

Comment:
Leave at least the brief part of the comment on each function, so readers don't have to jump to the header to know what the function does. Just don't put it in doxygen tags (No `/**` or `@brief`)

Reply:
What I do is take the contents of @brief from the header, and place it above the function in a regular old `//`.  Note doxygen ignores all comments that are not inside a oxygen comment indicator. Those are either `///` or `/** ... */`.

So if you don't have a triple slash or a double asterisk then doxygen ignores it.

Reply:
ok ill add something like this ```/* 
 * @brief Concatenates multiple gdf_columns into a single, contiguous column,
 * including the validity bitmasks.
 */```

Reply:
ugh the code thing isnt working but `/* */ style`